src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -121,6 +123,12 @@ export const EmptyStrat = {
   association: "",
   id: "",
 };
+export const DeleteStrat = {
+  cqlDefinition: "delete",
+  description: "delete",
+  association: "delete",
+  id: "",
+};
```

Comment:
A variable naming convention is usually a camel case, Correct me if I am wrong about it in TypeScript.